### PR TITLE
Added Fine branch

### PR DESCRIPTION
**Pull request by chessbyte**

### CODE REVIEW

**Comment by Fryguy** [comment]
This is missing the latest changes from master.

**Comment by Fryguy** [comment]
Specifically 

```yaml
  manageiq-content:
    has_rake_release: true
```

File: config/repos.yml [hunk 1]
```diff
@@ -13,6 +13,21 @@ master:
   manageiq-ui-classic:
   manageiq-ui-service:
   manageiq_docs:
+fine:
+  manageiq:
+    has_rake_release: true
+  manageiq-appliance:
+  manageiq-appliance-build:
+  manageiq-content:
```
Comment (Fryguy) [inline config/repos.yml]: This one here, needs the has_rake_release: true (which is present on latest master)

Comment (chessbyte) [inline config/repos.yml]: my bad.  updated

**Pull request merged**